dragonfly/entity/damage/source.go

```diff
@@ -45,10 +45,18 @@ type SourceFireTick struct{}
 // SourceLava is used for damage caused by being in lava.
 type SourceLava struct{}
 
+// SourceFall is a source that is used if the player fell.
+type SourceFall struct{}
+
 // SourceCustom is a cause used for dealing any kind of custom damage. Armour reduces damage of this source,
 // but otherwise no enchantments have an additional effect.
 type SourceCustom struct{}
 
+// ReducedByArmour ...
+func (SourceFall) ReducedByArmour() bool {
+	return true
```

Comment:
Does all armour reduce damage from fall damage, or just boots with feather falling?

Reply:
Armour doesn't reduce fall damage by itself.

Reply:
Would implementing reduced fall damage for feather falling be in the scope of this PR? Just wondering since things like PvP aren't affected by enchantments yet

Reply:
Nah not, I'd put that in a PR of a feather falling enchantment.

Reply:
Ah okay, this is resolved then.